docs/integration/telegraf.txt

```diff
@@ -0,0 +1,69 @@
+===============================
+Using Telegraf with CrateDB
+===============================
+
+`Telegraf <https://github.com/influxdata/telegraf>`_ is a plugin-driven server agent for collecting & reporting metrics.
+Unsing Telegraf in combination with CrateDB allows you collecting a huge amount of data and query them with SQL in real-time.
```

Comment:
can you make it more specific why crate is a good fit here. is telegraf not good at doing this by itself? why would you want to add crate to the mix?

Reply:
@nslater here are some additional reasons:
- Prometheus' internal storage has limited scalability.
- Users want to use the data outside of Prometheus to visualize it or run application code.
- They want to use CrateDB as long time storage.

Hope that's sufficient.

Reply:
@nslater Telegraf is a middleware connects inputs with outputs, so you may want to collect data with Kafka, Prometheus and what so ever and move them to what ever output like CrateDB. So you don't need to use CrateDB when using Telegraf. But you might have some input sources that you want to connect with CrateDB. Telegraf helps you get the data there. See this list https://github.com/influxdata/telegraf#input-plugins
You may also want to process or aggregate data before it is ingested into CrateDB.

Reply:
thanks @joemoe